### PR TITLE
feat(api): Add 1.5ml tuberack to old labware definition section

### LINKED ISSUE #2582: Add labware def for opentrons 4-in-1 tube rack: 1.5mL tubes
We have a 2mL tube+rack def , and a 15mL def, but not a 1.5mL def.

## Acceptance Criteria:

- [ ] Add def so you can `labware.load('opentrons-tuberack-1.5ml', slot)` in Python
- [ ] Migrate same def to `shared-data/definitions/opentrons-tuberack-1.5ml.json`
- [ ] Add to PD's list of addable labware (pair w/ Ian)

Then we need to get a beta release out so that people can use the labware on their robot.

## Note:

I don't know if periods `.` in labware load names are treated in some funny way, do we have any other labware with decimal points? Should we name it `1500uL` instead?

### PR DESCRIPTION
**Pull request by Laura-Danielle**
## overview
Closes #2582 

## changelog

- add 1.5ml definition in shared-data/definitions and default-containers

## review requests

If you can find 1.5ml tubes pls test on the robot.

### CODE REVIEW

File: shared-data/definitions/opentrons-tuberack-1.5ml-eppendorf.json [hunk 1]
```diff
@@ -0,0 +1,311 @@
+{
+  "metadata": {
+    "name": "opentrons-tuberack-2ml-eppendorf",
```
Comment (btmorr) [inline shared-data/definitions/opentrons-tuberack-1.5ml-eppendorf.json]: Disagrees with file name

Comment (Laura-Danielle) [reply shared-data/definitions/opentrons-tuberack-1.5ml-eppendorf.json]: Yup just saw that.

**Comment by btmorr** [comment]
Approved pending CI (I didn't look to see what is failing)

File: shared-data/definitions/opentrons-tuberack-1.5ml-eppendorf.json (hunk 1 above)
Comment (IanLondon) [reply shared-data/definitions/opentrons-tuberack-1.5ml-eppendorf.json]: Still needs fix!

**Pull request merged**